src/SDKs/DataFactory/Management.DataFactory/changelog.md

```diff
@@ -1,5 +1,11 @@
 # Changelog for the Azure Data Factory V2 .NET SDK
 
+## Version 0.7.0-preview
+
+### Feature Additions
```

Comment:
If our intention is to release a `0.7.0-preview` version, please update the csproj as well

Reply:
@dsgouda, please do not block this PR on that. I will update the version separately, that will include the csproj, description of changes and changelog.md

Reply:
In fact, can you approve this and I will run generate.cmd since I need to do those changes anyway?
It will update the files you want.

Reply:
@hvermis You are making some additive changes here, you need to at least update the csproj to `0.6.1-preview`

Reply:
Yes, I will do that. but I have to do it from my fork as I don't have permissions to this one. So asking you to merge this PR so I can update my fork and do those changes

Reply:
I don't think I can do that, I will need @shahabhijeet 's approval

Reply:
We usually do that - my other team members submit their changes, and then I update the version, the project file and changelog, and then publish the nuget. @shahabhijeet  is aware of our process. We need this change merged so I can publish the nuget ASAP and update the Powershell repo before code complete.